src/Orders/EmailAccounts/EmailAccount.php

```diff
@@ -74,4 +78,49 @@ public function delete(DeleteRequest $request): DeletedResponse
 
         return DeletedResponse::fromApiResponse($response);
     }
+
+    /**
+     * Makes a POST request to ResellerClub's 'update' email accounts.
+     *
+     * @see https://manage.resellerclub.com/kb/answer/1040
+     *
+     * @param UpdateDetailsRequest $request
+     * @return UpdateDetailsResponse
+     */
+    public function updateEmailDetails(UpdateDetailsRequest $request): UpdateDetailsResponse
```

Comment:
As we are in a class called `EmailAccount` why wouldn't you name this `update()` or `updateDetails()` or `modify()` (to match the endpoint)  is there any other relevant entity to differentiate from?